rascal-textmate-core/src/main/rascal/lang/oniguruma/Conversion.rsc

```diff
@@ -170,10 +170,14 @@ str encode(list[int] chars, bool withBounds = false)
     ? "\\b<encode(chars, withBounds = false)>\\b"
     : intercalate("", [encode(i) | i <- chars]);
 
-str encode(int char)
-    = char in alnum 
-    ? stringChar(char)
-    : "\\x{<toHex(char)>}";
+str encode(int char) 
+    = char in preEncoded
+    ? preEncoded[char]
+    : "\\x{<toHex(char)>}"
```

Comment:
```suggestion
    = preEncoded[char]
    ? "\\x{<toHex(char)>}"
```

(If that's indeed equivalent)

Reply:
Yes, I actually did it for performance reasons, but I shouldn't, unless it's slow.